factory.py

```diff
@@ -44,24 +47,22 @@ def create_from_verification_data(verification_data: VerificationData) -> arshin
     @staticmethod
     def __create_default() -> arshin.RecInfo:
         verification = arshin.RecInfo()
-        verification.sign_cipher = 'ДГХ'
         verification.mi_owner = 'ФИЗИЧЕСКОЕ ЛИЦО'
         verification.type = arshin.RecInfoType.VALUE_2
         verification.calibration = False
-        verification.doc_title = 'МИ 1592-2015'
         return verification
 
     @staticmethod
-    def __create_xmldate_from_string(string: str) -> XmlDate:
-        try:
-            meter_date = datetime.strptime(string, '%d.%m.%Y').date()
-        except ValueError:
+    def __create_xmldate_from_string(string: str, date_formats: list[str]) -> XmlDate:
+        for date_format in date_formats:
             try:
-                meter_date = datetime.strptime(string, '%d/%m/%Y').date()
+                meter_date = datetime.strptime(string, date_format).date()
```

Comment:
I'd recommend to place blank line before `return`, `yield`, `continue` and such statements if there's some code before it and if it doesn't contradict the project/language style guide.